wiktextract/data/fr/language_names_to_codes.json

```diff
@@ -5407,7 +5407,7 @@
   "Tokelauien": "tkl",
   "Tokharien A": "xto",
   "Tokharien B": "txb",
-  "Toki pona": "tokipona",
+  "Toki pona": "tok",
```

Comment:
This file is not used and can be deleted.

Reply:
Thanks for pointing this out. I deleted the file.